packages/shared-components/src/customComponents/README.md

```diff
@@ -0,0 +1,164 @@
+# Custom components and extending components
```

Comment:
Jeg lurer på om denne readme-en kanskje heller burde ligge under en /shared-components/docs/ mappe. Aller helst burde shared-components ha en README som refererer til denne. Og README på rot bør referere til README for hver enkelt pakke. Det er "way out of scope" her, og strengt tatt noe vi skulle ha rydda opp i etter monorepo-arbeidet.

Reply:
Foreslår en dokumentasjons-gjennomgang. Mye annet som bør dokumenteres også.
Lar det dermed ligge utenfor scope av denne oppgaven.